### PR TITLE
print version with git hash for simple version tracking

### PR DESCRIPTION
**Pull request by adamjmccann**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -59,8 +62,6 @@ PROXY_ARGS := $(shell if [ "$$http_proxy" != "" ]; then echo "-e http_proxy=$$ht
 PROXY_ARGS += $(shell if [ "$$https_proxy" != "" ]; then echo " -e https_proxy=$$https_proxy"; fi)
 PROXY_ARGS += $(shell if [ "$$no_proxy" != "" ]; then echo " -e no_proxy=$$no_proxy"; fi)
 
-INSTALL_DEPS = 1
```
Comment (jpward) [inline Makefile]: Did you mean to remove `INSTALL_DEPS`?  I believe this is here so we can get around Jenkins not building golint but still allow users to call it out.

Comment (adamjmccann) [reply Makefile]: oops, good catch James! Didn't mean to drop it, it's just miss in carrying the file from the internal repo. Putting it back now

Comment (adamjmccann) [reply Makefile]: ok, INSTALL deps is back

**Review by jpward — APPROVED** [review]
LGTM

**Pull request merged**